client/signup/config/flows.js

```diff
@@ -108,7 +108,7 @@ const Flows = {
 	defaultFlowName: config.isEnabled( 'signup/onboarding-flow' )
 		? abtest( 'improvedOnboarding' )
 		: 'main',
-	excludedSteps: [],
+	excludedSteps: new Set(),
```

Comment:
Nice use of `Set` here. You make, I learn!

Reply:
Would the Set be transpiled to something digestible by supported browsers p4TIVU-9ea-p2 ?

Reply:
React depends on `Set` and `Map`, so if they needed to be polyfilled they would have by now.
https://reactjs.org/docs/javascript-environment-requirements.html

Reply:
Just checked the project and it's already being used in a number of places 👍